doc/howto/write_a_view/solution_view.cpp

```diff
@@ -175,7 +175,7 @@ int main()
 
 //![main_adaptor]
     /* try the adaptor */
-    auto v2 = vec | std::view::reverse | view::my;
+    auto v2 = vec | std::view::reverse | ::view::my;
```

Comment:
?

Reply:
I honestly don't know why this test failed in my PR all of a sudden. but it did with `namespace view is ambigous`.

Reply:
:thinking: